inst/builtin/bs5/shiny/tables/_rules.scss

```diff
@@ -0,0 +1,95 @@
+:root {
+  --bslib-dt-row-selected: #{red($blue), green($blue), blue($blue)};
```

Comment:
```suggestion
  --bslib-blue-rgb: #{red($blue), green($blue), blue($blue)};
```

Reply:
Good idea! For now I'll just use `--bs-primary-rgb`, though, so the PR doesn't grow too much.